src/libraries/System.Security.Cryptography.Encoding/tests/Cbor.Tests/CborReaderTests.Integer.cs

```diff
@@ -100,6 +100,49 @@ public static void ReadCborNegativeIntegerEncoding_SingleValue_HappyPath(ulong e
             Assert.Equal(CborReaderState.Finished, reader.Peek());
         }
 
+        [Theory]
+        [InlineData(2, 2, "c202")]
+        [InlineData(0, "2013-03-21T20:04:00Z", "c074323031332d30332d32315432303a30343a30305a")]
+        [InlineData(1, 1363896240, "c11a514b67b0")]
+        [InlineData(23, new byte[] { 1, 2, 3, 4 }, "d74401020304")]
+        [InlineData(32, "http://www.example.com", "d82076687474703a2f2f7777772e6578616d706c652e636f6d")]
+        [InlineData(int.MaxValue, 2, "da7fffffff02")]
+        [InlineData(ulong.MaxValue, new object[] { 1, 2 }, "dbffffffffffffffff820102")]
+        public static void ReadTag_SingleValue_HappyPath(ulong expectedTag, object expectedValue, string hexEncoding)
+        {
+            byte[] encoding = hexEncoding.HexToByteArray();
+            var reader = new CborReader(encoding);
+
+            Assert.Equal(CborReaderState.Tag, reader.Peek());
+            ulong tag = reader.ReadTag();
+            Assert.Equal(expectedTag, tag);
```

Comment:
Do we already have a skip method?  If not, I think this is definitely where we need to add one.  E.g. "I don't know what to do with this tag, skip this value".  If it's a tagged tagged tagged map, how far do I have to jump as the caller? The reader should do that for me.

Reply:
There's no skip method yet. I was hoping to implement all major types before adding that.

Reply:
Since skip probably needs to vary its behavior on the data types, I guess that makes sense :)